packages/realm/src/Results.ts

```diff
@@ -104,6 +114,47 @@ export class Results<T = unknown> extends OrderedCollection<T> {
     }
   }
 
+  /**
+   * Add this query result to the set of active subscriptions. The query will be joined
+   * via an `OR` operator with any existing queries for the same type.
+   *
+   * @param options Options to use when adding this subscription (e.g. a name or wait behavior).
+   * @returns A promise that resolves to this {@link Results} instance.
```

Comment:
We should add `@experimental` to the API documentation

Reply:
Thanks! Forgot to add it to `subscribe()` as well